packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -257,7 +260,7 @@ describe("a mafs graph", () => {
             expect(points).toHaveLength(2);
         });
 
-        test("should render locked points with styles", async () => {
+        it("should render locked points with styles", async () => {
```

Comment:
😢 I don't think this needed to be changed, since `it` is an alias of `test`. No need to change it back now though.

Reply:
Agreed that it isn't necessary. I saw that we had a mix of uses (`it` vs `test`) and thought that I would clean things up for consistency.